### ISSUE TITLE
Link accessibility page to FAQ when it becomes available

### ISSUE THREAD

**Issue by hypha** [EuroPython/website#984]
In previous years, the accessibility has lots of information that is relevant to everyone. But not everyone checks the page when they try to find information. If we put similar type of info on the accessibility page this year, we should link it more prominently on the FAQ page to guide people there.

**Comment by nikoshell** [comment]
@hypha, is it the same page as last year? https://ep2024.europython.eu/accessibility/

**Comment by clytaemnestra** [comment]
@nikoshell the information about the main venue remains the same. In 2023, we had a more detailed accessibility page: https://ep2023.europython.eu/accessibility/. This year, we’re still waiting on the contract for the sprints venue, and the social events aren't finalized yet. Once everything is confirmed, I’d like to add accessibility info for the additional venues (sprints, socials) like we had in 2023.

I’d also prefer not to include a separate FAQ section on the page. Ideally, the content should be clear enough that an FAQ isn’t needed. Instead, we can link sections from the accessibility page to the general FAQ (at `/faq`).

**Comment by hypha** [comment]
completed in https://github.com/EuroPython/website/pull/1296 by Cyril
